presto-main/src/main/java/com/facebook/presto/operator/aggregation/AbstractMinMaxByNAggregationFunction.java

```diff
@@ -55,6 +56,7 @@
     private static final MethodHandle INPUT_FUNCTION = methodHandle(AbstractMinMaxByNAggregationFunction.class, "input", BlockComparator.class, Type.class, Type.class, MinMaxByNState.class, Block.class, Block.class, int.class, long.class);
     private static final MethodHandle COMBINE_FUNCTION = methodHandle(AbstractMinMaxByNAggregationFunction.class, "combine", MinMaxByNState.class, MinMaxByNState.class);
     private static final MethodHandle OUTPUT_FUNCTION = methodHandle(AbstractMinMaxByNAggregationFunction.class, "output", ArrayType.class, MinMaxByNState.class, BlockBuilder.class);
+    private static final long MAX_HEAP_CAPACITY = 10_000;
```

Comment:
I think `MAX_HEAP_CAPACITY` is to specific to implementation details. But I don't have a good suggestions (`MAX_N`, `MAX_NUMBER_OF_VALUES`? )

@haozhun